libs/core/testing/src/address/factories/address.factory.ts

```diff
@@ -0,0 +1,35 @@
+import { Injectable } from '@angular/core';
+import { Address, DaffodilAddress } from '@daffodil/core';
+
+import { ModelFactory } from '../../factories/factory.interface';
+import * as faker from 'Faker';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffAddressFactory implements ModelFactory<Address>{
+  
+  create(partial: Partial<Address> = {}) : MockDaffodilAddress {
+    return {
+      ...new MockDaffodilAddress(),
+      ...partial
```

Comment:
Would be cool if there was a test for creating a partially defined address, though it looks like you're probably going to add more tests later. Otherwise looks good so far